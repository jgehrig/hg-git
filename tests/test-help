#!/bin/sh

# Tests that the various help files are properly registered

echo "[extensions]" >> $HGRCPATH
echo "hggit=$(echo $(dirname $(dirname $0)))/hggit" >> $HGRCPATH

hg help | grep 'git' | sed 's/  */ /g'
hg help hggit | grep 'help git' | sed 's/:hg:`help git`/"hg help git"/g'
hg help git | grep 'Working with Git Repositories'
