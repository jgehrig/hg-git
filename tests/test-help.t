Tests that the various help files are properly registered

Load commonly used test logic
  $ . "$TESTDIR/testutil"

  $ hg help | grep 'git' | sed 's/  */ /g'
   hggit push and pull from a Git server
   git Working with Git Repositories
  $ hg help hggit | grep 'help git' | sed 's/:hg:`help git`/"hg help git"/g'
  For more information and instructions, see "hg help git"
  $ hg help git | grep 'Working with Git Repositories'
  Working with Git Repositories
