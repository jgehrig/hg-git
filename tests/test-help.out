 hggit push and pull from a Git server
 git Working with Git Repositories
For more information and instructions, see "hg help git"
Working with Git Repositories
